internal/email/generators.go

```diff
@@ -18,6 +18,11 @@ func GenerateGetPasswordReset(userName, password string) (string, error) {
 func GenerateRegistrationConfirmMessage(userName, websiteUrl, token string) (string, error) {
 	return generateHtml(generateRegistrationConfirmMessage(userName, websiteUrl, token))
 }
+
+func GenerateNewEmailConfirmMessage(userName, websiteUrl, token string) (string, error) {
```

Comment:
Please consider removing 'New' from the method name to align with other methods.